Makefile

```diff
@@ -189,8 +189,19 @@ payload: bots
 	bots/image-download $(PAYLOAD)
 
 .PHONY: images
-images: bots
-	bots/image-download $(TEST_OS) debian-stable ubuntu-stable fedora-41
+images: bots cloud-images iso-images
+
+.PHONY: cloud-images
+	bots/image-download debian-stable ubuntu-stable fedora-41
+
+.PHONY: iso-images
+	# If a compose if specified download from the compose otherwise
```

Comment:
I'm not sure this syntax works. I've always just seen `PHONY: <name>` as a declaration, not starting a rule. I.e. more like

```
.PHONY: images cloud-images iso-images

cloud-images:
          <commands>
```

etc.

If that combined phony/rule declaration works, then fine of course and TIL!

But also, do you really need three different rules here? Just make `images` download the stuff that you need?